scripts/create-example-tests.js

```diff
@@ -593,14 +593,24 @@ ${rows}
   ]);
 
   for (const row of refRows) {
-    refs[row.refId] = row.value.trim();
+    // assume refId and value column exists
+    const key = row.refId || row[0];
```

Comment:
Feels a bit hacky but not seeing where we could put the logic other than `readCSV`.
Any thoughts on the original issue here @mzgoddard?

Reply:
I think `typeof row.refId !== 'string'` should be treated as an error like with `row.value`.

Reply:
Hmm. Maybe instead of testing here it should be a separate function in the file scope like `validateCSVKeys`, that you add to the references csv promise chain? Maybe have it just perform tests and not modify refRows, then have the original for loop here?

and we can `.catch` on the `Promise.all(/* ... */)` call and `log.error` any error produced then.

The for loop here is legacy code for the older data file format. So I've been (probably overly) cautious in making changes to any such code.

```js
function validateCSVKeys({requiredKeys}, rows) {
  // test rows for keys ...
  // throw if a row is missing requiredKeys
  return rows;
}
```